src/acm_cert_retriever/retriever.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+
+"""Certificate Retriever."""
+import boto3
+import configargparse
+import logging
+import sys
+from Cryptodome.PublicKey import RSA
+from acm_common import logger_utils, truststore_utils
+
+
+logger = logging.getLogger("retriever")
+acm_client = boto3.client("acm")
+s3_client = boto3.client("s3")
+
+
+def parse_args(args):
+    """Parse the supplied command line arguments.
+
+    Returns:
+        argparse.NameSpace: The parsed and validated command line arguments
+
+    """
+    p = configargparse.ArgParser(
+        default_config_files=[
+            "/etc/acm_cert_helper/acm_cert_retriever.conf",
+            "~/.config/acm_cert_helper/acm_cert_retriever.conf",
+        ]
+    )
+
+    p.add(
+        "--acm-cert-arn",
+        required=True,
+        env_var="RETRIEVER_ACM_CERT_ARN",
+        help="ARN in AWS ACM to use to fetch the required cert, cert chain, and key",
+    )
+    p.add(
+        "--acm-key-passphrase",
+        required=True,
+        env_var="RETRIEVER_ACM_KEY_PASSPHRASE",
+        help="Passphrase to use to encrypt the downloaded key",
+    )
+    p.add(
```

Comment:
I think I'd prefer this to be:
```
p.add(
    "--add-downloaded-chain-to-truststore",
    action="store_true",
    env_var="RETRIEVER_ADD_DOWNLOADED_CHAIN"
    help=Add the downloaded cert chain from the ARN to the trust store)
```
Then the test in the code that does something with it can become the more Pythonic:

```
if args.add_downloaded_chain_to_truststore:
```

Reply:
Done!